### PR TITLE
Add bin directory to resin settings

### PR DESCRIPTION
**Pull request by CameronDiver**
The bin directory will be used to store binaries used by resin-cli,
among other pieces of infrastructure.

Change-type: minor
Signed-off-by: Cameron Diver <cameron@resin.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.6.2] - 2017-04-19
```
Comment (pimterry) [inline CHANGELOG.md]: Should be 3.6.0 really, and the `[ ]` are there because there should be a URL footnote at the bottom of this file that matches this.

Also, generally we don't do version number bumps on PRs for these modules. Process for these is typically:

- Make a change and just add a line to the changelog
- When you're ready to do an npm release:
  - Add the version number above whatever lines are included
  - Rebuild and check there's nothing missing from the build output
  - npm publish
  - git commit & tag & push to master

Basically I'm saying you should do the version part of this after this PR is merged. We don't want to end up with versions in the changelog that aren't on npm.

**Review by pimterry — CHANGES_REQUESTED** [review]
AppVeyor builds aren't passing because this repo still has 0.12 in the config, and modern npm doesn't support it any more. Can you just drop that from the appveyor config? That should fix the build.

Also there's no build output in here, you need to `npm build` and commit the result.

File: CHANGELOG.md (hunk 1 above)
Comment (CameronDiver) [reply CHANGELOG.md]: Ah gotcha, I've been spoiled by the VersionBot workflow :p. Will fix.

**Comment by CameronDiver** [comment]
Ping @pimterry amended.

File: CHANGELOG.md (hunk 1 above)
Comment (pimterry) [reply CHANGELOG.md]: Yep, we definitely need to start rolling that out to the npm modules soon! Not today though :-)

**Review by pimterry — APPROVED** [review]
